src/app/onion/core/learning-object.service.ts

```diff
@@ -434,21 +434,36 @@ export class LearningObjectService {
   setChildren(
     learningObjectName: string,
     authorUsername: string,
-    children: string[]
+    children: string[],
```

Comment:
If remove is true these are the children being removed right?

Reply:
Yes.  But there is really only one child removed at a time because there is a confirmation popup each time you remove a child